I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/Script/Chapter2/dedale.rpy", line 192, in script
    python:
  File "game/Script/Chapter2/dedale.rpy", line 193, in <module>
    if int(persistent.point)>=5:
TypeError: int() argument must be a string, a bytes-like object or a number, not 'NoneType'

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/Script/Chapter2/dedale.rpy", line 192, in script
    python:
  File "C:\Program Files (x86)\renpy-8.2.1-sdk\renpy\ast.py", line 823, in execute
    renpy.python.py_exec_bytecode(self.code.bytecode, self.hide, store=self.store)
  File "C:\Program Files (x86)\renpy-8.2.1-sdk\renpy\python.py", line 1178, in py_exec_bytecode
    exec(bytecode, globals, locals)
  File "game/Script/Chapter2/dedale.rpy", line 193, in <module>
    if int(persistent.point)>=5:
TypeError: int() argument must be a string, a bytes-like object or a number, not 'NoneType'

Windows-10-10.0.19045 AMD64
Ren'Py 8.2.1.24030407
A Programmer's Journey 0.1
Sat May  4 15:17:59 2024
